ElementX/Sources/Services/Analytics/Analytics.swift

```diff
@@ -31,18 +31,13 @@ import PostHog
 /// into `main`, update the AnalyticsEvents Swift package in `project.yml`.
 ///
 class Analytics {
-    /// The singleton instance to be used within the Riot target.
-    static let shared = Analytics()
-    
     /// The analytics client to send events with.
-    private var client: AnalyticsClientProtocol = PostHogAnalyticsClient()
-    
-//    /// The monitoring client to track crashes, issues and performance
-//    private var monitoringClient = SentryMonitoringClient()
-    
-    /// The service used to interact with account data settings.
-    private var service: AnalyticsService?
-    
+    private let client: AnalyticsClientProtocol
+
+    init(analytics: AnalyticsClientProtocol) {
```

Comment:
Nit: Probably clearer to init `Analytics` with a `client` than another `analytics`.
```suggestion
    init(client: AnalyticsClientProtocol) {
```